pdns/syncres.hh

```diff
@@ -460,7 +460,7 @@ public:
 
 private:
   struct GetBestNSAnswer;
-  int doResolveAt(set<DNSName> nameservers, DNSName auth, bool flawedNSSet, const DNSName &qname, const QType &qtype, vector<DNSRecord>&ret,
+  int doResolveAt(map<DNSName, pair<ComboAddress, bool> > nameservers, DNSName auth, bool flawedNSSet, const DNSName &qname, const QType &qtype, vector<DNSRecord>&ret,
```

Comment:
`nameservers` is modified in this function [https://github.com/PowerDNS/pdns/pull/3244/files#diff-9b3238d7a7cb2e313f1d8f1588971f93L1360](here)